riscv/src/compiler.rs

```diff
@@ -1247,6 +1315,23 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
                 rd,
             )
         }
+        "lh" => {
```

Comment:
Shouldn't we move that closer to lhu?

Reply:
I wanted to keep it close to "lb", but can be close to "lhu"